File: official/docs/python/v8/batches/add-shipments.py
import easypost
import os

client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))

batch = client.batch.retrieve("batch_...")

batch_with_shipments = client.batch.add_shipments(
    batch.id,
    shipments=[
        {"id": "shp_..."},
        {"id": "shp_..."},
    ],
)

print(batch_with_shipments)

File: official/docs/ruby/v6/batches/add-shipments.rb
require 'easypost'

client = EasyPost::Client.new(api_key: 'EASYPOST_API_KEY')

batch = client.batch.add_shipments(
  'batch_...',
  shipments: [
    { id: 'shp_...' },
    { id: 'shp_...' },
  ],
)

puts batch
